test/recipes/81-test_cmp_cli.t

```diff
@@ -248,6 +257,7 @@ sub load_tests {
     my $aspect = shift;
     my $test_config = $ENV{OPENSSL_CMP_CONFIG} // "$server_name/test.cnf";
     my $file = data_file("test_$aspect.csv");
+    my $tmpdir = bldtop_dir("test-runs/test_cmp_cli");
```

Comment:
In the same vein, maybe there should be a `result_dir` in OpenSSL::Test as well...

Reply:
Done, after introducing `result_dir` in OpenSSL::Test.